src/build_workflow/README.md

```diff
@@ -101,3 +102,15 @@ fi
 ```
 
 The [Jenkins workflows](../../jenkins) in this repository can use this mechanism to avoid rebuilding all of OpenSearch and OpenSearch Dashboards unnecessarily. 
+
+### Incremental Build
+
+This functionality augments the existing build process by introducing the `--incremental` binary parameter.
+
+Sample command: `./build.sh manifests/2.12.0/opensearch-2.12.0.yml --incremental`.
+
+The build workflow will examine the build manifest from the previous build using the example path `tar/builds/opensearch/manifest.yml` when this command is executed.
+The build workflow will be executed in accordance with the comparison between the commits for each component in the preceding build manifest and the current input manifest.
+It will contain every modified component, and every component that relies on these revised components based on the `depends_on` entry in the input manifest.
+
+Once build is finished, a new build manifest will be generated using the previous build manifest as a reference, ensuring that all non-modified components remain unchanged.
```

Comment:
I think the the build manifest path is constant? `{distribution}/builds/opensearch/manifest.yml` Better to mention it that it will look for build manifest in that location.

Also worth mention that new artifacts will overwrite the previously built artifacts as well as manifest.

Reply:
Updated. Thanks.